llvm/lib/Analysis/VectorUtils.cpp

```diff
@@ -23,6 +23,7 @@
 #include "llvm/IR/Constants.h"
 #include "llvm/IR/DerivedTypes.h"
 #include "llvm/IR/IRBuilder.h"
+#include "llvm/IR/Intrinsics.h"
```

Comment:
This shouldn't be necessary b\c IRBuilder.h already includes Intrinsics.h. I'm assuming a tool added it so feel free to ignore.